docs/getting-started/quickstart.md

```diff
@@ -39,108 +40,100 @@ Let's create an index configured to receive Wikipedia articles.
 
 ```bash
 # First, download the Wikipedia config from Quickwit repository.
-curl -o wikipedia_index_config.json https://raw.githubusercontent.com/quickwit-inc/quickwit/main/examples/index_configs/wikipedia_index_config.json
+curl -o wikipedia_index_config.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/tutorials/wikipedia/index-config.yaml
 ```
 
 The index config defines three text fields: `title`, `body` and `url`. It also sets two default search fields `body` and `title`. These fields will be used for search if you do not target a specific field in your query. Please note that by default, text fields are [indexed and tokenized](../reference/index-config.md).
 
 And here is the complete config:
 
-```json title="wikipedia_index_config.json"
-{
-    "default_search_fields": ["body", "title"], // If you do not specify fields in your query, those fields will be used. 
-    "field_mappings": [
-        {
-            "name": "body",
-            "type": "text"
-        },
-        {
-            "name": "title",
-            "type": "text"
-        },
-        {
-            "name": "url",
-            "type": "text",
-            "indexed": false, // Field not indexed, you will not be able to search on this field.
-            "stored": false  // Field not stored. 
-        }
-    ]
-}
+```yaml title="wikipedia_index_config.yaml"
+version: 0
+
+doc_mapping:
+  field_mappings:
+    - name: title
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+      stored: true
+    - name: url
+      type: text
+      tokenizer: raw
+      stored: true # Field is stored. 
+
+search_settings:
+  # If you do not specify fields in your query, those fields will be used.
+  default_search_fields: [title, body]
 ```
 
-Now we can create the index with the command:
+Let's make use of the default configuration file provided in quickwit installation throughout this guide.
```

Comment:
"by setting the" starts a new sentence, it seems that it's the continuation of the previous.